### PR TITLE
Fixes #75: Add Watchnet DVR support

### LINKED ISSUE #75: Add WatchNet DVR
WatchNet is based on Dahua.

OUI is `52:54:4c`

The tested RTSP url is:
`rtsp://admin:1234@192.168.1.97:554/cam/realmonitor?channel=05&subtype=0`

So the pattern is:
`rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}&subtype={{stream}}`

16 `channels` are `01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16`

2 `streams` are `0;1` where 0 is the best (but only D1 resolution 240p) and 1 is nearly unusable lo-res so be sure to default to `0`.

username is `admin`
default password is `1234`

### PR DESCRIPTION
**Pull request by jhallard**
I'm not sure of the exact model number this is for so I just put the
name as 'Watchnet DVR' and used a user manual for *a* WatchNet DVR,
@jcmaslan if you lmk the model you have access to I can change the entry
to reflect such.

### CODE REVIEW

File: cameras/paths.csv [hunk 1]
```diff
@@ -35,3 +35,4 @@ Grandstream,nvr,GXV3504,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address
 Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}},H.264,554,4,,Stream {{stream}},,4,4,admin,admin,,http://www.grandstream.com/sites/default/files/Resources/GXV3674hd_fhd_v2_usermanual_english.pdf
 Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}}.sdp,H.264,554,live;live2;liveany,,Stream 1;Stream 2;Stream 3,,live2,live,,,,http://download.vivotek.com/downloadfile/downloads/usersmanuals/cc8130manual_en.pdf
 Dahua,camera,HFW81200,(4C:11:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}},H.264,554,,,,,,,admin,admin,,http://www1.dahuasecurity.com/products/ipc-hfw81200e-z-1159.html
+Watchnet,nvr,Watchnet DVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://watchnetinc.com/public/pdfs/VMS%20Lite%20Users%20Manual_Ver1.0.pdf
```
Comment (jcmaslan) [inline cameras/paths.csv]: WatchNet is the brand's capitalization.
Missing brace on channel.

**Review by jcmaslan — CHANGES_REQUESTED** [review]
missing mustache brace

File: cameras/paths.csv (hunk 1 above)
Comment (jhallard) [reply cameras/paths.csv]: 0a114b4

**Comment by jcmaslan** [comment]
LGTM

**Comment by jhallard** [comment]
I'll merge this but we should try and close change-requests officially before merging in the future. It'd be nice if Github counted 'LGTM' comments as a sign of a change-request being resolved like they interpret 'Fixes #XX' as a sign of an issue being resolved.

**Comment by jhallard** [comment]
Deployed to testing and production, /paths output for WatchNet key now:

```json
john@hostname: ~/Cameras   (john)
$ curl https://camio-test-cameras.appspot.com/paths | jq '.WatchNet'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 20930  100 20930    0     0  45135      0 --:--:-- --:--:-- --:--:-- 45205
[
  {
    "rtsp_server": "rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}",
    "model": "WatchNet DVR",
    "default_values": {
      "username": {
        "is_multiselect": false,
        "options": [
          {
            "value": "admin"
          }
        ]
      },
      "password": {
        "is_multiselect": false,
        "options": [
          {
            "value": "1234"
          }
        ]
      },
      "port": {
        "is_multiselect": false,
        "options": [
          {
            "value": "554"
          }
        ]
      },
      "stream": {
        "is_multiselect": false,
        "options": [
          {
            "name": "High",
            "value": "0"
          },
          {
            "name": "Low",
            "value": "1"
          }
        ]
      },
      "channel": {
        "is_multiselect": true,
        "options": [
          {
            "name": "Channel 01",
            "value": "01"
          },
          {
            "name": "Channel 02",
            "value": "02"
          },
          {
            "name": "Channel 03",
            "value": "03"
          },
          {
            "name": "Channel 04",
            "value": "04"
          },
          {
            "name": "Channel 05",
            "value": "05"
          },
          {
            "name": "Channel 06",
            "value": "06"
          },
          {
            "name": "Channel 07",
            "value": "07"
          },
          {
            "name": "Channel 08",
            "value": "08"
          },
          {
            "name": "Channel 09",
            "value": "09"
          },
          {
            "name": "Channel 10",
            "value": "10"
          },
          {
            "name": "Channel 11",
            "value": "11"
          },
          {
            "name": "Channel 12",
            "value": "12"
          },
          {
            "name": "Channel 13",
            "value": "13"
          },
          {
            "name": "Channel 14",
            "value": "14"
          },
          {
            "name": "Channel 15",
            "value": "15"
          },
          {
            "name": "Channel 16",
            "value": "16"
          }
        ]
      }
    },
    "rtsp_path": "/cam/realmonitor?channel={{channel}}&subtype={{stream}}",
    "maker": "WatchNet"
  }
]
```

**Pull request merged**